webmachine/models.py

```diff
@@ -50,7 +50,7 @@ class Token(models.Model):
     verifier = models.CharField(max_length=VERIFIER_SIZE)
     consumer = models.ForeignKey(Consumer,
             related_name="tokens_consumer")
-    timestamp = models.IntegerField(default=time.time())
```

Comment:
this one made my date :dart: each time `makemigrations` will create a new migration